nicknamer.py

```diff
@@ -61,8 +61,12 @@ async def nick(context: Context, member: Member, new_nickname: str) -> None:
     Args:
         context: The discord `Context` from which the command was invoked
         member: Member whose nickname should be changed
-        new_nickname: New nickname which should be applied to `member`
+        new_nickname_input: New nickname which should be applied to `member`, may be
+                            list of strings, which will be joined to form a single
+                            string nickname
     """
+    new_nickname: str = " ".join(new_nickname_input)
```

Comment:
Check if an empty string or a space leads to unfortunate results here, other than that lgtm

Reply:
I don't think space could be an input. Respecting empty strings would make the nickname exactly what is typed in, so that's fine. The input from the discord message is already parsed by splitting on spaces; that's why I had to do this feature in the first place!